missioncontrol/etl/tasks.py

```diff
@@ -18,3 +22,35 @@ def update_measures():
             for measure_name in platform['measures']:
                 update_measure.apply_async(
                     args=[platform_name, channel_name, measure_name])
+
+
```

Comment:
Just a nit: The names of these 2 tasks sound very similar. Something like `update_experiment_data` seems to fit and describe what the task is doing.